enums/MicrosoftAudience.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/**
+ * @file enums/MicrosoftAudience.php
+ *
+ * Copyright (c) 2024 Simon Fraser University
+ * Copyright (c) 2024 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @class MicrosoftAudience
+ *
+ * @brief Enumeration for Microsoft supported Audience
+ */
+
+namespace APP\plugins\generic\openid\enums;
+
+enum MicrosoftAudience: string
+{
+    case COMMON = 'common';
+    case CONSUMERS = 'consumers';
+    case ORGANIZATIONS = 'organizations';
+
+    public static function toArray(): array
```

Comment:
These functions look like they might be good candidates for a general-purpose abstraction... and at a glance, it looks like there's already an implementation of that here!

https://github.com/laracraft-tech/laravel-useful-additions/blob/main/src/Traits/UsefulEnums.php